man/subscription-manager.8

```diff
@@ -649,7 +649,7 @@ Identifies the organization for which to list the configured environments.
 .SS REPOS OPTIONS
 The
 .B repos
-command lists all of the repositories that are available to a system. This command is only used for organizations which have a locally-hosted content service of some kind, like Subscription Asset Manager. With Red Hat's hosted content service, there is only one central repository.
+command lists all of the repositories that are available to a system.  With Red Hat's hosted content service, there is only one central repository.
```

Comment:
I think that second sentence "With Red Hat's hosted content service, there is only one central repository."  is not correct too. Red Hat's CDN provides thousands of repositories. @cnsnyder, what do you think?

Reply:
Yea I agree. There are many repositories. I think we can just simplify this to "This command can list all repositories available to a system. It can also enable and disable access to those repositories". Or something to that effect.